fastlane/Fastfile

```diff
@@ -428,6 +429,59 @@ platform :android do
     )
   end
 
+  desc 'Builds a prototype build and uploads it to S3'
+  lane :build_and_upload_prototype_build do |options|
+      UI.user_error!("'BUILDKITE_ARTIFACTS_S3_BUCKET' must be defined as an environment variable.") unless ENV['BUILDKITE_ARTIFACTS_S3_BUCKET']
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/9fb120d7cb9100f73700cfa1412673f9ce6dd3e1/fastlane/Fastfile#L434"></span>Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.</td>
    </tr>
  </tbody>
</table>